### DISCUSSION TITLE
How about customizable switches or security system status ?

### DISCUSSION THREAD

**Discussion by SebbySullivan** [MiguelRipoll23/homebridge-securitysystem#160] [Ideas]
Hi ! First of all I wanted to thank the developper for this great plugin that's been really useful for me !

I was wondering if the Homekit security system status could be customized or if Apple made a restriction so that the different modes had to be "Home" "away" and "night" ? I'm pretty sure they did because those modes are automatically translated when you use iOS/macOS/iPadOS in another language (which I do since I'm French.. Excuse my English btw)

I'm asking this because I'd need a "Vacation mode": I have pets, a huge Husky and a little cat who often trigger the motion detectors in my house (even though Aqara and Philips sell them to be "pet proof"). That's why I set automations that trigger the SS (Security system) only when door opening sensors are triggered. 
But when I leave to go on vacation I bring my pets with me and I would like to reinforce the system by adding the motion detectors to the triggering sensors (In case thieves would enter by the windows or by digging a tunnel underneath my house xD). That's why I thought of a "vacation" mode I would only have to activate to add the motions sensors. 
Right now, I use a switch I added using the "Dummy switch" homebridge plugin which I called "Vacation mode" and my automations are set with the following condition: "When a motion detector is triggered, if the SS is set to AWAY and if the Vacation Mode switch is ON THEN trigger the security system alarm"
But I'm a little bit fussy by nature and it bothers me to have a switch, separated from the SS button. So I told myself, maybe an option could be added to the plugin to customize the different status or to add customizable switches. 
I tried another way: using the Mode Away Extend Switch and rename it "Vacation mode" into HK but it's directly linked to the Away status and it automatically sets the Away mode when it's activated. But I'd like to be able to set the vacation mode maybe 1 or 2 days ahead when I think of it, just to make sure I won't forget and when I leave and my automations set the Away mode, the vacation switch is already ON.

I hope my post is understandable, it's quite hard to explain myself in a foreign language :p 
Thank you for reading my post, and thank you for this plugin !

**Comment by SebbySullivan** [comment]
As i said at the end of my post I already tried this. But it’s not perfect. Just thought it would be cool to be able to add customized switches :)

**Reply by MiguelRipoll23** [reply]
My bad, misread the end (still recovering from the vaccine). One thing you could try is to turn on the mode away extended switch automatically when you’re far away from your house or when you reach your destination. The former can be done using the Home app creating an automation, would that help?

**Reply by MiguelRipoll23** [reply]
Hey @SebbySullivan, did the previous comment help?
